packages/react/src/themes/teams/components/Toolbar/toolbarMenuDividerVariables.ts

```diff
@@ -0,0 +1,3 @@
+import toolbarVariables from './toolbarVariables'
+
+export default toolbarVariables
```

Comment:
I am not sure that I like this pattern because all variables are defined in a single place will affect others 🤔

Reply:
This is now consistent with `Menu`. Let's discuss it separately and eventually refactor both.